src/main/java/javax/validation/constraints/Past.java

```diff
@@ -64,6 +78,11 @@
 	Class<? extends Payload>[] payload() default { };
 
 	/**
+	 * @return true if the present date should be considered valid
+	 */
+	boolean orPresent() default false;
```

Comment:
Yes.

We build a corresponding `java.time` object from the `Clock` so to validate a `Year`, we build a `Year` object with the `Clock`. Thus 2016 would be valid for every day in 2016.